tests/catalog/data/small_sky/partition_info.csv

```diff
@@ -0,0 +1,2 @@
+order,pixel,num_objects
+0,11,131
```

Comment:
It looks like partition_info.csv and catalog_info.json have some common data. How do we ensure that they remain compatible so that some future agent might change one and not the other?

Reply:
There's some common info with partition_info and <catalog_name>_info.json. The <catalog_name>_info.json is used with some earlier prototying data, and the new ideal has the info split up into two files: partition_info.csv and catalog_info.json. I'd like the older format to go away once we've validated the newer formats.